OpenRA.StyleCheck/OpenRA.StyleCheck.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
VS added a BOM here (the others have it, so this adds consistency). I also allowed it to trim the trailing newlines.